src/Commands/MakeFactoryReloadedCommand.php

```diff
@@ -32,24 +40,45 @@ class MakeFactoryReloadedCommand extends GeneratorCommand
     /** @var string */
     private $className;
 
+    /** @var string */
+    private $modelsPath;
+
+    /** @var string */
+    private $modelFile = '';
```

Comment:
Any reason why this is set to an empty string?

Reply:
Oops. That's a remnant of me mindlessly hacking around last week trying to get it solved without you having to set that method to public for me.